content/docs/getting_started/images/_index.md

```diff
@@ -0,0 +1,10 @@
++++
```

Comment:
I don't believe that an index is necessary in the `images` folder.. though I see that they all have them 😕

Reply:
It is 👀  It couldn't find the directory without it